cypress/e2e/SignIn.cy.ts

```diff
@@ -14,3 +17,16 @@ describe('Name and Email Validation', () => {
     cy.signInByMailAndCheck(GRAASP);
   });
 });
+
+describe('Already signed in', () => {
+  beforeEach(() => {
+    cy.setUpApi({ currentMember: MEMBERS.BOB });
+  });
+  it('Should show redirection content', () => {
+    cy.visit('/');
+    cy.get(`#${REDIRECTION_CONTENT_CONTAINER_ID}`);
+    cy.get(`[role="button"]`).click();
+    cy.get('h1').should('contain', 'Content');
```

Comment:
Reuse text var?

Reply:
This tests the text that is returned by the static mock. Should I add it in a constant ?